src/types/mercuryEvents.ts

```diff
@@ -0,0 +1,156 @@
+import { MercuryAuth } from './auth'
```

Comment:
based on our naming, this file would be `src/types/events.types.ts` (Don't pay attention to highlight line 1 of this file, not sure how to comment on just the name)